bin/birdhouse

```diff
@@ -63,8 +65,105 @@ Example Usage:
+                                                                                                       [--birdhouse-logs] 
+                                                                                                       [--local-env-file]
+                                                                                                       [--no-clobber]"
+BACKUP_RESTORE_USAGE=$(echo $BACKUP_RESTORE_USAGE | tr "\n" " ")
+
+BACKUP_HELP="$BACKUP_USAGE
+
+Manage backups for the birdhouse stack.
+
+Commands:
+  create:     Create a backup for data used by the birdhouse stack.
+  restore:    Restore data from a backup snapshot.
+  restic:     Run a command using the restic CLI (birdhouse uses restic to manage backups).
+
+Options:
+  -h, --help    Print this message and exit
+"
+
+BACKUP_CREATE_HELP="$BACKUP_CREATE_USAGE
+
+Create a backup for data used by the birdhouse stack.
+
+Options:
+  -h, --help                          Print this message and exit
+  -d, --dry-run                       Print information about which data will be backed up. Do not actually perform backup.
+  -a, --component-application-data    Backup application data for a given component (e.g. magpie). This can be specified multiple times. Use '*' to select all components.
+  -u, --component-user-data           Backup user data for a given component (e.g. cowbird). This can be specified multiple times. Use '*' to select all components.
+  -l, --component-logs-data           Backup logs data for a given component (e.g. thredds). This can be specified multiple times. Use '*' to select all components.
+  --birdhouse-logs                    Backup birdhouse logs and docker container logs
+  --local-env-file                    Backup local environment file
+  --no-restic                         Do not save this backup as a restic snapshot. This will only backup the data to the directory specified by the BIRDHOUSE_BACKUP_DIR variable.
+
+Note:
+  Not all components store application application data, user data, or log data. Depending on the component, some or none of these
+  will be available to backup. Use the --dry-run flag to confirm which data will be backed up.
+
+Example Usage:
+
+  $ birdhouse backup create -a '*' -u '*' -l '*' --birdhouse-logs --local-env-file # backs up everything
+  $ birdhouse backup create -a magpie # backs up application data for the magpie component
+  $ birdhouse backup create -a cowbird -u cowbird -a finch # backs up application data for cowbird and finch and the user data for cowbird
+"
+
+BACKUP_RESTORE_HELP="$BACKUP_RESTORE_USAGE
+
+Restore data from a backup snapshot.
+
+Options:
+  -h, --help                          Print this message and exit
+  -s, --snapshot                      ID of the snapshot that should be recovered. To list available snapshots run: ${THIS_BASENAME} backup restic snapshots
```

Comment:
This command is note the same as below example `birdhouse restic snapshots` (no `backup`).
Is this one or the other the correct one?

Also, consider adding the full command between quotes.